cliquet/cache/__init__.py

```diff
@@ -85,6 +86,7 @@ def ping(request):
                 backend.set(_HEARTBEAT_KEY, 'alive', _HEARTBEAT_TTL_SECONDS)
             return True
         except:
+            logger.exception("Heartbeat Failure")
```

Comment:
In both cases we get the same info about the error - But I like to use explicit messages because it makes it easier to grep in the logs. 

e.g. "Q: why is the heartbeat down. ?  A: let me grep "heartbeat failures" in the logs.


```python
>>> try:
...     raise Exception('pok')
... except Exception as e:
...     logging.exception(e)
...
ERROR:root:pok
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
Exception: pok

>>> try:
...     raise Exception('pok')
... except:
...     logging.exception("Heartbeat Error")
...
ERROR:root:Heartbeat Error
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
Exception: pok
```